sample-apps/javascript-sample-app/server.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS'" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+'use strict';
+
+// setting up the traces and metrics
+const tracer = require('./tracer');
+const meter = require('./meter');
+
+const http = require('http');
+const AWS = require('aws-sdk');
+const Worker = require('worker_threads');
+
+const { updateTotalBytesSent, updateLatencyTime } = require('./request-metrics');
+
+const api = require('@opentelemetry/api');
+const create_cfg = require('./config');
+const { DiagConsoleLogger } = require('@opentelemetry/api');
+
+function startServer(file='./config.yaml') {
+    const worker = new Worker.Worker('./random-metrics.js');
+    const cfg = create_cfg.create_config(file);
+    const server = http.createServer(handleRequest);
+    server.listen(cfg.Port, cfg.Host, (err) => {
+      if (err) {
+        throw err;
+      }
+      console.log(`Node HTTP listening on ${cfg.Host}:${cfg.Port}`);
+    });
+}
+
+function handleRequest(req, res) {
+    const cfg = create_cfg.create_config('./config.yaml');
```

Comment:
It shouldn't be necessary to do this on every request.  The config is loaded once in `startServer()` and should be made available to this code from there.

Reply:
Thanks. Updated.